src/tasks/dailyItems.ts

```diff
@@ -289,5 +290,17 @@ export const DailyItemTasks: Task[] = [
       },
       choices: { 1280: 1 },
     },
+    {
+      name: "Lodestone",
+      ready: () => have($item`lodestone`) && !get("_lodestoneUsed"),
+      completed: () => get("_lodestoneUsed"),
+      do: () => use($item`lodestone`),
+    },
+    {
+      name: "Update Garbage Tote",
+      ready: () => have($item`January's Garbage Tote`) && !get("_garbageItemChanged"),
+      completed: () => get("_garbageItemChanged"),
+      do: () => cliExecute("fold broken champagne bottle"),
+    },
```

Comment:
There are 2 solutions to this. Either we update the garbage tote state here at the start of day if it's not updated, or we could check for `!get("_garbageItemChanged") || get("garbageChampagneCharge") > 0` for the drum machines and `!get("_garbageItemChanged") || get("garbageShirtCharge") > 0` for NEP.

I'm more inclined towards this solution since this means we don't have to remember to add the condition to every piece of code that uses the tote.